variables.tf

```diff
@@ -52,6 +52,10 @@ variable "bastion" {
         memory      = "16"
         processors  = "1"
     }
+    validation {
+        condition       = lookup(var.bastion, "count", 1) >= 1 && lookup(var.bastion, "count", 1) <= 3
+        error_message   = "The bastion.count value must be a valid integer between 1 to 3."
```

Comment:
I'm wondering do we really need 3? Since it's just active/passive we can restrict the count to 2. Thoughts ?

Reply:
Even I was wondering what should be the ideal value. After evaluating the technical limitation on no of external IP addresses in pub-vlan I have concluded 3 can be done,

But I guess yes maximum `2` make sense with less cost and management. I will change it to 2 but if anyone asks for a new CR we can just update the number here.. the code is made to handle N no of bastion nodes.

Reply:
makes sense. Let's have 2 as max and if we really get a usecase for more than that it can be changed